rental.rb

```diff
@@ -0,0 +1,16 @@
+require 'date'
+
+class Rental
+  attr_accessor :date
```

Comment:
- [ ] [OPTIONAL] you can add the `attr_accessor` for `:data` in the same line as `:person` and `:book`

Reply:
Sorry, I asked for a code review without fixing the errors. I have already made these changes.
Thanks for the review 😃